From: Eric Evans <eevans@debian.org>
Date: Sun, 29 Nov 2015 12:33:59 -0600
Subject: do-not-use-vendored-code

Upstream botocore includes a couple of vendored modules, and boto3 depends
on them transitively.  The Debian package of botocore excludes this vendored
code, and includes dependencies for the corresponding packages, so we need
to do the same, here.
---
 boto3/s3/transfer.py              | 3 +--
 tests/functional/test_dynamodb.py | 2 +-
 tests/unit/s3/test_transfer.py    | 2 +-
 3 files changed, 3 insertions(+), 4 deletions(-)

Index: python-boto3/boto3/s3/transfer.py
===================================================================
--- python-boto3.orig/boto3/s3/transfer.py
+++ python-boto3/boto3/s3/transfer.py
@@ -135,8 +135,7 @@ import boto3
 from concurrent import futures
 
 from botocore.compat import six
-from botocore.vendored.requests.packages.urllib3.exceptions import \
-    ReadTimeoutError
+from requests.packages.urllib3.exceptions import ReadTimeoutError
 from botocore.exceptions import IncompleteReadError
 
 import boto3.compat
Index: python-boto3/tests/functional/test_dynamodb.py
===================================================================
--- python-boto3.orig/tests/functional/test_dynamodb.py
+++ python-boto3/tests/functional/test_dynamodb.py
@@ -13,7 +13,7 @@
 import json
 from tests import unittest, mock
 
-from botocore.vendored import requests
+import requests
 
 from boto3.session import Session
 from boto3.dynamodb.conditions import Attr
Index: python-boto3/boto3/compat.py
===================================================================
--- python-boto3.orig/boto3/compat.py
+++ python-boto3/boto3/compat.py
@@ -14,8 +14,7 @@ import sys
 import os
 import errno
 import socket
-
-from botocore.vendored import six
+import six
 
 if six.PY3:
     # In python3, socket.error is OSError, which is too general
Index: python-boto3/tests/unit/s3/test_transfer.py
===================================================================
--- python-boto3.orig/tests/unit/s3/test_transfer.py
+++ python-boto3/tests/unit/s3/test_transfer.py
@@ -20,7 +20,7 @@ from contextlib import closing
 import mock
 from botocore.stub import Stubber
 from botocore.session import Session
-from botocore.vendored import six
+import six
 from concurrent import futures
 
 from boto3.exceptions import RetriesExceededError
